trezor-crypto/crypto/bip39.c

```diff
@@ -46,23 +46,28 @@ CONFIDENTIAL struct {
 
 #endif
 
-const char *mnemonic_generate(int strength) {
```

Comment:
`mnemo` can we comment it out? to avoid confusion since it's not used

Reply:
done

Reply:
0xa36fD8fF0524f287422337d99bABdf05923f581B